gensrc/thrift/StatusCode.thrift

```diff
@@ -100,6 +100,8 @@ enum TStatusCode {
     TXN_NOT_EXISTS = 51,
     TXN_IN_PROCESSING = 52,
 
-    RESOURCE_BUSY = 53
+    RESOURCE_BUSY = 53,
+
+    AUTOMATIC_PARTITION = 54
```

Comment:
```suggestion
    LATE_PARTITION = 54
```
like `LATE MATERIALIZATION`

Reply:
It is different from the semantics of `LATE`, I changed it to `AUTOMATIC_PARTITION_IN_PROCESSING`